tests/app/celery/test_process_ses_receipts_tasks.py

```diff
@@ -331,6 +331,7 @@ def test_ses_callback_should_set_status_to_permanent_failure(client,
     assert get_notification_by_id(notification.id).status == 'sending'
     assert process_ses_receipts_tasks.process_ses_results(ses_hard_bounce_callback(reference='ref'))
     assert get_notification_by_id(notification.id).status == 'permanent-failure'
+    assert 'Failed' in get_notification_by_id(notification.id).status_reason
```

Comment:
I would test for the complete error message on line 197.

Reply:
> Can you link to any logs, perhaps in dev, that shows this working as intended?

You can search for "bounce" in the dev-api logs from yesterday.
Here's a [link](https://console.amazonaws-us-gov.com/cloudwatch/home?region=us-gov-west-1#logsV2:log-groups/log-group/dev-notification-api-log-group/log-events$3Fstart$3D1669870800000$26end$3D1669957199000$26filterPattern$3D$2522bounce$2522)